htdocs/GetCSS.php

```diff
@@ -93,6 +90,20 @@
     $FullPath = "$ModuleDir/css/$File";
 }
 
+$public = false;
+try {
+    $m = Module::factory($Module);
+
+    $public = $m->isPublicModule();
+} catch(LorisModuleMissingException $e) {
+    $public = false;
+}
+if ($anonymous === true && $m->isPublicModule() === false) {
```

Comment:
this needs to refer to the variable `$public` created above, `$m` may not exist in this scope if an exception was thrown.